plugins/language_clojure.lua

```diff
@@ -0,0 +1,53 @@
+-- Author: https://github.com/RohanVashisht1234/
+-- mod-version:3
+local syntax = require "core.syntax"
+syntax.add {
+  name = "Clojure",
+  comment = ";;",
+  files = {
+    "%.clojure$",
+  },
+  patterns = {
+    { pattern = ';;.*',                                   type = 'comment' },
+    { pattern = ';.*',                                    type = 'comment' },
+    { pattern = { '#"', '"', '\\' },                      type = 'string' },
+    { pattern = { '"', '"', '\\' },                       type = 'string' },
+    { pattern = { '"""', '"""', '\\' },                   type = 'string' },
+    { pattern = 'Retention%s+()[%a_][%w_/]*',             type = { 'keyword', 'literal' } },
+    { pattern = ':[%a_][%w_/%-]*',                        type = { 'keyword2', 'literal' } },
+    { pattern = '[%a_][%w_]*()%.()[%a_][%w_/%-]*',        type = { 'keyword','operator', 'keyword2' } },
+    { pattern = "%(()def()%s+()[%a_][%w_%-]*",            type = { "normal", "keyword", "literal", 'literal' } }, -- tested ok
+    { pattern = "%(()def[%a_][%w_]*()%s+()[%a_][%w_%-]*", type = { "normal", "keyword", "literal", 'literal' } }, -- tested ok
```

Comment:
Is this about things that start with `def`? So even things like `def_whatever`?

Reply:
yes, anything that starts with def is to be highlighted